src/vs/base/browser/ui/iconLabel/iconLabel.ts

```diff
@@ -255,7 +319,7 @@ class LabelWithHighlights {
 				this.singleLabel = new HighlightedLabel(dom.append(this.container, dom.$('a.label-name', { id: options?.domId })), this.supportCodicons);
 			}
 
-			this.singleLabel.set(label, options?.matches, options?.title, options?.labelEscapeNewLines);
+			this.singleLabel.set(label, options?.matches, undefined, options?.labelEscapeNewLines);
```

Comment:
Was this intentional because we now set the hover over the entire label?

Reply:
Yes, if we don't pass undefined here then we can end up with a native tooltip too.